apps/console/src/features/connections/components/edit/forms/helpers/form-fields-helper.tsx

```diff
@@ -617,7 +659,9 @@ export const getFieldType = (
     propertyMetadata: CommonPluggableComponentMetaPropertyInterface,
     mode: AuthenticatorSettingsFormModes
 ): FieldType => {
-    if (propertyMetadata?.type?.toUpperCase() === CommonConstants.BOOLEAN) {
+    if (propertyMetadata?.key === ConnectionManagementConstants.USER_ID_IN_CLAIMS) {
+        return FieldType.RADIO;
+    } if (propertyMetadata?.type?.toUpperCase() === CommonConstants.BOOLEAN) {
```

Comment:
Should this be a `else if`?
If not add a newline here

Reply:
Improved with 5047722b4b25f0551dd0da7ca417f606a3ce039a.